src/AdminConsole/Pages/Organization/Admins.cshtml.cs

```diff
@@ -133,22 +133,25 @@ public async Task<IActionResult> OnPostInvite(InviteForm form)
 
     public async Task<IActionResult> OnPostCancel(string hashedCode)
     {
-        Invites = await _invitationService.GetInvitesAsync(User.GetOrgId().Value);
+        Invites = await _invitationService.GetInvitesAsync(User.GetOrgId()!.Value);
```

Comment:
Refactor this possibly so it doesn't require the organisation id to be passed. It's a good opportunity to touch this now.

Reply:
@jonashendrickx If I reccall correctly, there's reason why we pass it down instead of through injection because the invitationService doesn't always run in a org-context.

Reply:
I'm going to mark this as resolved and we can tackle it later if necessary.